solution-base/mongodb/Makefile

```
@@ -5,11 +5,10 @@ CHART_REPO:="https://charts.bitnami.com/bitnami"
 CHART_MONGO_SHARDED_VERSION:="7.9.1"
 
 PATCH_DIR:="${ROOT_DIR}/patches"
-PATCH_FILES:="$(shell ls -d ${PATCH_DIR}/*)"
-
+PATCH_FILES:=$(shell ls ${PATCH_DIR}/* | grep -v mongodb-sharded-add-configsvr-service-file.patch)
```

should not filter patches: if we need not to apply it, then we must actually remove it (`git rm`)...

```suggestion
PATCH_FILES:=$(shell ls ${PATCH_DIR}/*)
```